source/2019-04-05-the-ember-times-issue-92.md

```diff
@@ -29,6 +29,18 @@ Curious to learn more? Check out her post on Medium: [3 reasons why you should v
 
 ---
 
+## [Animations and Predictable Data Loading in Ember 🐹](https://crunchingnumbers.live/2019/04/02/animation-and-predictable-data-loading-in-ember/)
+
+[@ijlee2](https://github.com/ijlee2) a recent attendee at [Ember Conf 2019](https://emberconf.com/) shares an [tutorial](https://crunchingnumbers.live/2019/04/02/animation-and-predictable-data-loading-in-ember/) on how to load complex data in a predictable manor and how to use animations to liven up your ember applications. 
```

Comment:
Can we say `[@ijlee2](https://github.com/ijlee2), a first-time attendee at [EmberConf](https://emberconf.com/), shares a [tutorial]()`? I hope that this may inspire other first-time attendees to write about their experience and/or learning.